src/lib.rs

```diff
@@ -160,7 +166,12 @@ pub struct Config {
 
     /// The hyper builder for the internal http client.
     #[serde(skip)]
-    pub hyper_builder: hyper::client::Builder,
+    #[serde(default = "default_builder")]
+    pub hyper_builder: hyper_util::client::legacy::Builder,
```

Comment:
what is the right way to get rid of legacy?

Reply:
There is no replacement at the moment. The client has some issues which is why it has been moved out of hyper to hyper-util. A new client will be built at some point.